modules/virt-editing-cdi-cpu-and-memory-defaults.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-cdi-for-namespace-resourcequota.adoc
+
+[id="virt-editing-cdi-cpu-and-memory-defaults_{context}"]
+= Editing the `CDIConfig` object to override CPU and memory defaults
+
+Modify the default settings for CPU and memory requests and limits for your
+use case by editing the spec attribute of the `CDIConfig` object.
+
+.Procedure
+
+. Use the `oc` client to edit the `cdiconfig/config` and edit the
```

Comment:
In this step, you're just doing the first part, so this should stop at "Use the `oc` client to edit the `cdiconfig/config`:" 

Most of our procedure steps have phrasing more like this, though: "Edit the `cdiconfig/config` by running the following command:"

You can also add a `.Prerequisites` section to this module that says:
* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.